### ISSUE TITLE
Visited link colour is not shown correctly after app restart

### ISSUE THREAD

**Issue by valynor** [brave/brave-ios#8523]
Brave Version 1.60 (23.11.27.16) BraveCore 1.60.122 (119.0.6045.163)
Device iPad11,1 (iPadOS 17.1.2), iPad mini 5

Brave iOS does not show the correct colour for visited links after the app gets terminated and restarted. 

Example: 
1. open https://en.wikipedia.org/wiki/Main_Page (or really ANY other website with links)
2. tap any link, let it load
3. swipe left to go back: the link colour changed from blue to purple as it should be
4. close the tab
5. exit Brave and remove from task switcher
6. open Brave again, go back to https://en.wikipedia.org/wiki/Main_Page and find that all of the purple links are blue (unvisited) again
7. looking at the history DOES show the visited URLs from before

In short: while the Brave app stays open the visited link colours behave as expected and show visited links in purple but as soon as you close and terminate Brave, all previously visited links are shown in blue (unvisited) again - apparently across all websites. 

The iPad is in a sync chain with 3 Brave desktops, doing the same on Brave desktop works fine. Visited URLs get synced across devices, too.

**Comment by iccub** [comment]
I checked other 3rd party browsers and all of them show the same behavior. So it is most likely a WKWebView limitation. Closing as wontfix for now
